lib/_tls_wrap.js

```diff
@@ -345,7 +345,7 @@ function initRead(tlsSocket, socket) {
 
   // Socket already has some buffered data - emulate receiving it
   if (socket && socket.readableLength) {
-    var buf;
+    let buf;
```

Comment:
nit: make buf const and move it to line 349
```javascript
while ((const buf = socket.read()) !== null)
```

Reply:
@trivikr like this? not sure if it's possible
![Screenshot 2019-11-07 at 00 46 00](https://user-images.githubusercontent.com/4695397/68340738-12788880-00f8-11ea-8525-c7199a36249a.png)

Reply:
You're right - it's not possible.